frontend_vue/src/components/item/ShowItem.vue

```diff
@@ -86,12 +86,16 @@ export default {
     onCopyButtonClick() {
       const { item, $copyText, $buefy } = this;
       $copyText(item.value).then(() => {
-        $buefy.dialog.alert({
-          message: `"${item.value}" copied to clipboard :)`,
-          type: "is-primary",
-          ariaRole: "alertdialog",
-          ariaModal: true,
-        });
+        if ((item.value).includes('http://') || (item.value).includes('https://')) {
```

Comment:
I believe would be nice to extract this to a method like `isLink()` and if an item is a link the icon should also change to something that represents "open a link", currently we are using icons from font awesome.

Reply:
Also, please add specs to these scenarios, the specs should test:
1) if the user is properly redirected when click button link
2) if the icon has changed from "copy icon" to "open link icon" when  the value is a link
3)  if the icon has not changed from "copy icon" to "open link icon" when  the value is not a link

Reply:
closed by https://github.com/edimossilva/memory_app/pull/65